src/init.jl

```diff
@@ -1,4 +1,7 @@
-using LinearAlgebra, SparseArrays
+using IterTools, LinearAlgebra, SparseArrays
 
 using MultivariatePolynomials: AbstractVariable, AbstractMonomialLike,
-      exponents, variables, powers, monomials, coefficient
\ No newline at end of file
+      exponents, variables, powers, monomials, coefficient
+
+using LazySets: Hyperrectangle, dim, low, high
+using IntervalArithmetic: interval
```

Comment:
If this is the only thing that is needed, then I suggest to instead use the `Interval` from `LazySets` to reduce the amount of packages involved.